test_scripts/API/AutoCompleteList/011_ATF_SetGlobalProperties_N_AutoCompleteList_TabInValue.lua

```diff
@@ -0,0 +1,59 @@
+--------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL SDL respond <success = false, resultCode = "INVALID_DATA"> to mobile app
+--
+-- Description:
```

Comment:
@Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..."
2. in case INVALID_DATA is expected, SDL should not transfer request to HMI at all - please update description properly